server/plugin.go

```diff
@@ -145,6 +145,16 @@ func (p *Plugin) UserHasLeftTeam(pluginContext *plugin.Context, tm *model.TeamMe
 }
 
 func (p *Plugin) MessageHasBeenPosted(pluginContext *plugin.Context, post *model.Post) {
+	shouldProcessMessage, err := p.Helpers.ShouldProcessMessage(post, plugin.BotID(p.api.Configurator.GetConfig().BotUserID))
+	if err != nil {
+		p.mm.Log.Error("Error while checking if the message should be processed %v", "err", err.Error())
```

Comment:
The message here will be shown as `Error while checking if the message should be processed %v`, since no formatting is done to this string (the key value passed will just add another field to the log message, not do any kind of formatting). I would remove the %v.

Reply:
Good catch @larkox :+1: